### PR TITLE
DDF-3397 Documenting the profile:install command

### PR DESCRIPTION
**Pull request by brjeter**
#### What does this PR do?
Documents the new profile:install command.
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@oconnormi @figliold 
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@lessarderic 
@ricklarsen - Documentation
#### What are the relevant tickets?
[DDF-3397](https://codice.atlassian.net/browse/DDF-3397)
#### Checklist:
- [X] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+:type: installing
+:status: published
+:summary: Installing from ${command-console}.
+:project: ${branding}
+:order: 08
+
+===== Completing Installation from the ${command-console}
+
+In order to install ${branding} from the ${command-console}, use the command `profile:install <profile-name>`.
+The `<profile-name>` should be the desired <<_setup_types, Setup Type>> in lowercase letters.
```
Comment (figliold) [inline distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc]: :question: Should you mention that there is a profile:list command, so an admin can see all of the available profiles?

File: distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+[NOTE]
+====
+This only installs the desired Setup Type. There are other components that can be set up in the ${admin-console} Installer that cannot be setup on the ${command-console}.
```
Comment (figliold) [inline distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc]: ❓ @ricklarsen , what is the general rule for links?  I see there is a link for Setup Type above but not here. Do we make a link for every occurrence?

File: distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc]: I like this idea

File: distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc (hunk 2 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc]: Not for every occurrence. I'd say as a rule of thumb, it depends on how close the occurrences are to each other. If there is some distance between this one and the last one, it makes sense to link them.

File: distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc (hunk 1 above)
Comment (brjeter) [reply distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc]: Addressed

File: distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc (hunk 2 above)
Comment (brjeter) [reply distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc]: Yeah, these occurrences are really close together so I think I'll just link on the first reference.

File: distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+===== Completing Installation from the ${command-console}
+
+In order to install ${branding} from the ${command-console}, use the command `profile:install <profile-name>`.
+The `<profile-name>` should be the desired <<_setup_types, Setup Type>> in lowercase letters.
+To see the available profiles, use the command `profile:list`.
+
+[NOTE]
+====
+This only installs the desired Setup Type. There are other components that can be set up in the ${admin-console} Installer that cannot be setup on the ${command-console}.
+After installing the Setup Type, these other components can be set up as prescribed below.
```
Comment (lessarderic) [inline distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc]: ✏️ `described` instead of `prescribed`?

**Comment by brjeter** [comment]
All comments addressed

**Comment by brjeter** [comment]
build now

**Pull request merged**